### ISSUE TITLE
"Send Test Email to ..." button for a lesson generates PHP warnings / notices

### ISSUE THREAD

**Issue by AiratHalitov** [woocommerce/sensei-content-drip#131]
`Send Test Email to ...` button in lesson settings is not working.

Every time when I push the button I see an error in WooCommerce logs.

### Error Log:
```
2018-02-20T03:21:05+00:00 CRITICAL Uncaught ArgumentCountError: Too few arguments to function Scd_Ext_Drip_Email::send_single_email_drip_notifications(), 2 passed in /my_path/wp-content/plugins/sensei-content-drip/includes/class-scd-ext-manual-drip.php on line 333 and exactly 3 expected in /my_path/wp-content/plugins/sensei-content-drip/includes/class-scd-ext-drip-email.php:292
Stack trace:
#0 /my_path/wp-content/plugins/sensei-content-drip/includes/class-scd-ext-manual-drip.php(333): Scd_Ext_Drip_Email->send_single_email_drip_notifications('8', Array)
#1 /my_path/wp-includes/class-wp-hook.php(286): Scd_Ext_Manual_Drip->send_test_email('')
#2 /my_path/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters('', Array)
#3 /my_path/wp-includes/plugin.php(453): WP_Hook->do_action(Array)
#4 /my_path/wp-admin/admin-ajax.php(97): do_action('wp_ajax
```

### Configuration:
- Sensei Content Drip 1.0.8
- Sensei 1.9.20
- BuddyPress 2.9.3
- WooCommerce 3.3.1 -> 3.3.2
- WordPress 4.9.4

**Comment by aheckler** [comment]
Can repro.

**Comment by AiratHalitov** [comment]
Thanks!
Do you know when will be new version of plugin?

**Comment by donnapep** [comment]
The email was sent, but several PHP warnings and notices were logged that need to be fixed.

**Comment by AiratHalitov** [comment]
Still actual: 

```
2018-05-10T14:27:16+00:00 CRITICAL Uncaught ArgumentCountError: Too few arguments to function Scd_Ext_Drip_Email::send_single_email_drip_notifications(), 2 passed in /my_path/wp-content/plugins/sensei-content-drip/includes/class-scd-ext-manual-drip.php on line 333 and exactly 3 expected in /my_path/wp-content/plugins/sensei-content-drip/includes/class-scd-ext-drip-email.php:292
Stack trace:
#0 /my_path/wp-content/plugins/sensei-content-drip/includes/class-scd-ext-manual-drip.php(333): Scd_Ext_Drip_Email->send_single_email_drip_notifications('8', Array)
#1 /my_path/wp-includes/class-wp-hook.php(286): Scd_Ext_Manual_Drip->send_test_email('')
#2 /my_path/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters('', Array)
#3 /my_path/wp-includes/plugin.php(453): WP_Hook->do_action(Array)
#4 /my_path/wp-admin/admin-ajax.php(97): do_action('wp_ajax
```

**Comment by jenhooks** [comment]
I have a user reporting this in 1824037-zen. @donnapep -- could you check out the PR @aheckler submitted, and see if it can be merged? https://github.com/woocommerce/sensei-content-drip/pull/132

**Comment by donnapep** [comment]
@AiratHalitov Would appreciate a quick test to ensure this is fixed in the [2.0 beta](https://github.com/woocommerce/sensei-content-drip/releases/tag/version%2F2.0.0-beta.2). Thx!

**Comment by AiratHalitov** [comment]
@donnapep 
I can't because I'm not using this plugin. `¯\_(ツ)_/¯`
